lib/lightning_web/live/job_live/job_builder_components.ex

```diff
@@ -9,53 +11,20 @@ defmodule LightningWeb.JobLive.JobBuilderComponents do
   attr :source, :string, required: true
   attr :rest, :global
 
+  jsx("assets/js/job-editor/JobEditorMain.tsx")
+
   def job_editor_component(assigns) do
     assigns = assigns |> assign(disabled: assigns.disabled |> to_string())
 
     ~H"""
-    <div
-      data-job-id={@job_id}
-      data-adaptor={@adaptor}
-      data-source={@source}
-      data-disabled={@disabled}
-      data-disabled-message={@disabled_message}
-      data-change-event={@change_event}
-      phx-hook="JobEditor"
-      phx-update="ignore"
+    <.JobEditorMain
```

Comment:
Yeah, here's where the component name is particularly tricky. hm.

Reply:
This name has to match what the mounting React component is called. Hence if we change the mounting react component name to `JobEditor` then we can simple change this too to that.